exercises/practice/circular-buffer/.docs/instructions.append.md

```diff
@@ -1,7 +1,5 @@
 # Instructions append
 
-## Tasks
```

Comment:
This removes it from a practice exercise. Is that intended?

Reply:
It wasn't. Sorry about that.

Reply:
Fixed

Reply:
No worries, I was just wondering if it might have been a change to the spec too. Thanks for fixing it :)